examples/flow.cpp

```diff
@@ -171,11 +228,22 @@ try
         return EXIT_FAILURE;
     }
 
-    // Grid init
     std::vector<double> porv = eclipseState->getDoubleGridProperty("PORV")->getData();
-    grid.reset(new GridManager(eclipseState->getEclipseGrid(), porv));
-    auto &cGrid = *grid->c_grid();
-    const PhaseUsage pu = Opm::phaseUsageFromDeck(deck);
+#if USE_DUNE_CORNERPOINTGRID
+    // Dune::CpGrid as grid manager
+    typedef Dune::CpGrid  Grid;
+    std::shared_ptr<Grid> gridPtr;
```

Comment:
Since grid creation is in the main() scope, we can simplify a little by not going via the shared pointer but using a stack variable.